shared/src/crypto.rs

```diff
@@ -103,6 +103,8 @@ pub trait GetCerts: Sync + Send {
     async fn certificate_list_via_network(&self) -> Result<Vec<String>, SamplyBeamError>;
     async fn certificate_by_serial_as_pem(&self, serial: &str) -> Result<String, SamplyBeamError>;
     async fn im_certificate_as_pem(&self) -> Result<String, SamplyBeamError>;
+    /// A callback that runs on a timer and returns if the cache changed
+    async fn on_timer(&self, _cache: &mut CertificateCache) -> bool { false }
```

Comment:
Consider using an enum rather than a bool to make it more explicit and extensible for future uses.

Reply:
Done